src/components/Dashboard/TodoList.vue

```diff
@@ -1932,100 +1937,81 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
 
       case FilingTypes.INCORPORATION_APPLICATION: {
         // navigate to Create UI to resume this Incorporation Application
         const incorpAppUrl = `${this.getCreateUrl}?id=${this.tempRegNumber}`
         navigate(incorpAppUrl)
-        break
+        return
       }
 
       case FilingTypes.REGISTRATION: {
         // navigate to Create UI to resume this Registration
         const registrationAppUrl = `${this.getCreateUrl}define-registration?id=${this.tempRegNumber}`
         navigate(registrationAppUrl)
-        break
+        return
       }
 
       case FilingTypes.ALTERATION: {
         // navigate to Edit UI to resume this Alteration
         const alterationUrl = `${this.getEditUrl}${this.getIdentifier}/alteration/?alteration-id=${item.filingId}`
         navigate(alterationUrl)
-        break
+        return
       }
 
       case FilingTypes.DISSOLUTION: {
         // navigate to Create UI to resume this Dissolution
         const dissolutionUrl = `${this.getCreateUrl}define-dissolution?id=${this.getIdentifier}`
         navigate(dissolutionUrl)
-        break
+        return
       }
 
       case FilingTypes.CHANGE_OF_REGISTRATION: {
         // navigate to Edit UI to resume this Change of Registration
         const changeUrl = `${this.getEditUrl}${this.getIdentifier}/change/?change-id=${item.filingId}`
         navigate(changeUrl)
-        break
+        return
       }
 
       case FilingTypes.CONVERSION: {
         // navigate to Edit UI to resume this Conversion
         const conversionUrl = `${this.getEditUrl}${this.getIdentifier}/conversion/?conversion-id=${item.filingId}`
         navigate(conversionUrl)
-        break
+        return
       }
 
       case FilingTypes.SPECIAL_RESOLUTION: {
         // navigate to Edit UI to resume this Special Resolution
         const specialResolutionUrl =
           `${this.getEditUrl}${this.getIdentifier}/special-resolution/?special-resolution-id=${item.filingId}`
         navigate(specialResolutionUrl)
-        break
+        return
       }
 
       case FilingTypes.RESTORATION:
         this.navigateForResumeRestoration(item)
-        break
-
-      default:
-        // eslint-disable-next-line no-console
-        console.log('doResumeFiling(), invalid type for item =', item)
-        break
+        return
     }
 
-    function navigateToCorrectionEditUi (editUrl: string, identifier: string): void {
-      // resume correction via Edit UI
-      const correctionUrl = `${editUrl}${identifier}/correction/?correction-id=${item.filingId}`
-      navigate(correctionUrl)
-    }
-
-    function routeToLocalCorrection (router: VueRouter): void {
-      // resume local correction
-      router.push({
-        name: Routes.CORRECTION,
-        params: { filingId: item.filingId.toString(), correctedFilingId: item.correctedFilingId.toString() }
-      })
-    }
+    // eslint-disable-next-line no-console
+    console.log('doResumeFiling(), invalid type item =', item)
```

Comment:
In case we try to resume a draft correction that shouldn't exist (ie, was created before this code was implemented).